app/assets/stylesheets/govuk-component/_breadcrumbs.scss

```diff
@@ -15,4 +17,30 @@
     color: $secondary-text-colour;
     text-decoration: none;
   }
+
+  @include media-down(mobile) {
+    &.collapse-on-mobile li {
+      display: none;
+
+      &.parent-breadcrumb {
+        background-image: none;
+        display: block;
+        margin-left: 0;
+        padding-left: 14px;
+        position: relative;
+
+        &:before {
+          border-bottom: 5px solid transparent;
+          border-right: 6px solid $back-link-arrow-color;
```

Comment:
If this is the current text color can we remove this entirely and rely on the browsers `currentColor` behaviour?

https://developer.mozilla.org/en/docs/Web/CSS/color_value#currentcolor_keyword

This would look like `border-right: 6px solid;'